src/test/java/org/fundacionjala/sfdc/stepdefinition/OpportunityStepsCheck.java

```diff
@@ -4,7 +4,8 @@
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
-import org.fundacionjala.sfdc.pageobjects.SalesForceTabClassic;
+import org.fundacionjala.sfdc.pageobjects.SalesForceObject;
+import org.fundacionjala.sfdc.pageobjects.SalesForceSection;
 import org.fundacionjala.sfdc.pageobjects.opportunities.OpportInputs;
 import org.fundacionjala.sfdc.pageobjects.opportunities.SFODetailsPage;
 import org.fundacionjala.sfdc.pageobjects.opportunities.SFOMainPage;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.opportunities.SFOMainPage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)